apps/export/tests/test_apis.py

```diff
@@ -56,3 +56,17 @@ def test_trigger_api_with_export_permission(self):
         export = Export.objects.get(id=response.data['export_triggered'])
         self.assertTrue(export.pending)
         self.assertEqual(export.exported_by, self.user)
+
+    def test_export_filter_by_status(self):
```

Comment:
We will also need a test where a client archives an export.
Also, @AdityaKhatri Can a user unarchive an export?